src/pages/lesson-plans/lesson-06.html

```diff
@@ -337,26 +337,26 @@ <h2 class="main">Integrated Lesson # 6</h2>
 <p>1. Introduction/Review (5 minutes)</p>
 <ol>
 <li>
-<step><p>Review rules quickly.</p>
+<step [stepsContext]="steps" [highlightedStepId]="suggestedStepId" [currentQuestionStepId]="gettingResponsesFor"><p>Review rules quickly.</p>
```

Comment:
Here is the start of a Step component -- it's also just a draft, but I've thought it through a bit more.  As I start talking about implementation details, they're just what I envisioned... but that doesn't mean I've thought it all the way through. Feel free to change or whatever as needed.


 This is an example of a step which is "just" a step to be toggled (not a question with responses to be gathered).  Even so, it has some assumptions about things that exist inside the LessonPlan component, because they are passed to Step inputs.

- `steps` is an array of Step components, which would be created empty within LessonPlan and then passed to the steps themselves so they can sort of "register" themselves with the parent LessonPlan.  I feel like there might be a better way to approach this in terms of proper Angular2+ architecture, but this was the most straightforward way I could think of.
- `suggestedStepId` is the stepId of the first step after the last completed step, as described (with slightly different words) in the ticket
- `gettingResponsesFor` is either null, or the stepId of the question for which responses are currently being gathered